scripts/manage-isos.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -u
+
+usage()
+{
+    echo "----------------------------------------------------------------------------------------"
+    echo "- Usage: $0 [version XCP-ng] [Repository] [Server PXE] [Path on the server PXE] [Server SR ISO] [Path on the server of the SR ISO] [Path on the pXE for th installer]"
+    echo "- All options are mandatory."
+    echo "- Repository: updates, ci, testing..."
+    echo "- Version of XCP-ng: 8.2 or 8.3"
+    echo "- Example: bash manage-isos.sh 8.3 myuser@my_pxe /my/path/pxe/ iso_user@srv-sr-iso /my/path/on/sriso/ /my/path/pxe/installers/"
+    echo "----------------------------------------------------------------------------------------"
+}
```

Comment:
A short description of what the command does would be great, especially since "manage" is a very generic word